src/main/resources/db/migration/V1_0__create_audit_event_table.sql

```diff
@@ -0,0 +1,10 @@
+create table audit_event
+(
+    id              uuid primary key,
+    operation_id    varchar(80),
+    what            varchar(200) not null,
+    occurred        timestamp with time zone not null,
+    who             varchar(80),
+    service         varchar(200),
+    details         varchar(1000)
```

Comment:
and this is where you want to try a JSON type right?

Reply:
yes, the details column is just declared as "json" rather than "varchar(1000)"